test/runtests.jl

```diff
@@ -26,7 +26,7 @@ max_worker_rss != typemax(Csize_t) && move_to_node1("parallel")
 cd(dirname(@__FILE__)) do
     n = 1
     if net_on
-        n = min(8, Sys.CPU_CORES, length(tests))
```

Comment:
Perhaps people doing that should use a command line parameter? Or alternately, there should be an easier way to use more cores for tests.